### PR TITLE
Fix indenting on cache check in Network layer class

### PR DESCRIPTION
**Pull request by footh**
Resolves [#14054](https://github.com/tensorflow/tensorflow/issues/14054)

The `else` and ensuing code on the cache check was indented one spot too far. This resulted in the cache hit code not being executed when there was a cache hit (and the cache miss code never being called because there was a cache hit).

### CODE REVIEW

**Review by fchollet — APPROVED** [review]
LGTM

**Comment by martinwicke** [comment]
I apologize that this has languished for so long.

This code has moved to network.py. Could you rebase?

@fchollet, should we make this fix internally?

**Comment by footh** [comment]
Looking at the conflict (lines 2125-2135), it looks like this issue has already been resolved. So this PR is probably moot at this point.

**Comment by footh** [comment]
Oops, no, I take that back. I just reread your comment and see the bug is in the network.py class. I will rebase.

File: tensorflow/python/layers/base.py [hunk 1]
```diff
@@ -1358,7 +1358,6 @@ def __repr__(self):
                                                          self.get_shape(),
                                                          self.dtype.name)
 
-
```
Comment (martinwicke) [inline tensorflow/python/layers/base.py]: I think we want this blank line.

**Review by martinwicke — CHANGES_REQUESTED** [review]

**Review by martinwicke — APPROVED** [review]

File: tensorflow/python/layers/base.py (hunk 1 above)
Comment (martinwicke) [reply tensorflow/python/layers/base.py]: Never mind.

**Comment by martinwicke** [comment]
Thanks! Testing now.

**Comment by yifeif** [comment]
@tensorflow-jenkins test this please

**Pull request merged**